marshmallow_enum/__init__.py

```diff
@@ -51,7 +51,7 @@ def __init__(
         if load_by is None:
             load_by = LoadDumpOptions.value if by_value else LoadDumpOptions.name
 
-        if load_by not in LoadDumpOptions:
+        if not isinstance(load_by, Enum) or load_by not in LoadDumpOptions:
```

Comment:
The aim was to avoid this warning (shown with pytest >= 3.8.0):

```py

DeprecationWarning: using non-Enums in containment checks will raise TypeError in Python 3.8
  if load_by not in LoadDumpOptions:
```

Reply:
Ah, didn't realize that was a change coming. Carry on. 👌